indico/modules/events/sessions/util.py

```diff
@@ -153,3 +154,64 @@ def get_sessions_for_user(event, user):
                         ~Session.is_deleted)
                 .all())
     return {sess for sess in sessions if any(user in entry.principal for entry in iter_acl(sess.acl_entries))}
+
+
+def serialize_session_for_ical(sess):
+    session_data = {
+        '_fossil': 'sessionMetadata',
+        'id': sess.id,
+        'startDate': sess.start_dt,
+        'endDate': sess.end_dt,
+        'url': '',
+        'title': sess.title,
+        'location': sess.venue_name,
+        'roomFullname': sess.room_name,
+        'description': ''
+    }
+
+    description = ''
+    for c in sess.contributions:
+        if c.speakers:
+            speakers = ('{} ({})'.format(s.person.full_name, s.person.affiliation) for s in c.speakers)
+            description += 'Speakers: {}\n'.format(', '.join(speakers))
+
+    session_data['description'] = description
+    return session_data
+
+
+def serialize_session(sess):
+    """Return data for a single session"""
+    from indico.modules.events.util import serialize_person_link
+
+    def _serialize_date(dt):
+        tzinfo = sess.event_new.tzinfo
+        return {'date': dt.astimezone(tzinfo).strftime('%Y-%m-%d'),
+                'time': dt.astimezone(tzinfo).strftime('%H:%M:%S'),
+                'tz': str(tzinfo)}
+
+    data = {
+        '_fossil': 'session',
```

Comment:
was it like this before? if not just get rid of it; it's not for the API and if no JS code depends on it we don't need it